pkgs/tools/networking/pdsh/default.nix

```diff
@@ -12,6 +12,13 @@ stdenv.mkDerivation rec {
   buildInputs = [ perl readline ssh ]
     ++ (lib.optional slurmSupport slurm);
 
+  nativeBuildInputs = [ autoreconfHook ];
+
+  # Do not use git to derive a version.
+  postPatch = ''
+    sed -i 's/m4_esyscmd(\[git describe.*/[${version}])/' configure.ac
```

Comment:
This postPatch hunk seems unrelated to the commit topic. Split into a new commit?